jekyll.gemspec

```diff
@@ -7,7 +7,7 @@ Gem::Specification.new do |s|
   s.specification_version = 2 if s.respond_to? :specification_version=
   s.required_rubygems_version = Gem::Requirement.new(">= 0") if s.respond_to? :required_rubygems_version=
   s.rubygems_version = "2.2.2"
-  s.required_ruby_version = ">= 2.0.0"
+  s.required_ruby_version = ">= 2.1.0"
```

Comment:
OK so Jekyll 4 can bump Ruby dependencies from 2.0 to 2.4 for instance, but Jekyll 3.5 cannot bump from 2.0 to 2.1, even if [2.0 is not tested anymore on CI](https://github.com/jekyll/jekyll/blob/master/.travis.yml#L8-L10).

PR updated accordingly.